app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue

```diff
@@ -3,18 +3,112 @@
     <div class="gobierto-data-sql-editor">
       <SQLEditorHeader />
       <SQLEditorCode />
+      <SQLEditorTabs
+        v-if="data"
+        :items="data"
+        :link="link"
+        :active-tab="activeTabIndex"
+        @active-tab="activeTabIndex = $event"
+      />
     </div>
   </div>
 </template>
 <script>
+import axios from 'axios';
```

Comment:
Is SQLEditor a *container*? Has any sense they manage data? If so, I shut up. But I'm seeing a lot of `this.$root` listeners all over the code that make me suspicious.